catalog/app/containers/Bucket/Queries/QueryViewer.tsx

```diff
@@ -1,13 +1,28 @@
+import Ajv from 'ajv'
+import brace from 'brace'
+import { JsonEditor } from 'jsoneditor-react'
 import * as React from 'react'
 import * as M from '@material-ui/core'
+import * as Lab from '@material-ui/lab'
 
-import JsonEditor from 'components/JsonEditor'
+import 'brace/mode/json'
+import 'brace/theme/eclipse'
+import 'jsoneditor-react/es/editor.min.css'
+
+import schema from 'schemas/query.json'
 
 import * as requests from './requests'
 
+import '../../../../static/json-editor.css'
```

Comment:
if the file is in `static` dir, it gets copied to the build dir and expected to be loaded from html; but if you want to import css via webpack, you should just place it somewhere in the app, like, maybe inside the current component dir